app/src/main/java/com/example/pix/chat/ChatFragment.java

```diff
@@ -81,10 +90,33 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         ivPictures = view.findViewById(R.id.chat_pictures);
         ivNewPic = view.findViewById(R.id.chat_image);
 
+        // If we click the camera, go to the ComposeFragment
+        ivCamera.setOnClickListener(view12 -> {
```

Comment:
I meant to call it "view1". I don't actually use it, its a parameter that is given when your view is clicked. I couldn't name them "view" because "view" was already taken in the scope.